includes/class-convertkit-api.php

```diff
@@ -440,10 +440,12 @@ public function get_resource( $url ) {
 
 				// Maybe inflate response body.
 				// @see https://wordpress.stackexchange.com/questions/10088/how-do-i-troubleshoot-responses-with-wp-http-api
+				/*
```

Comment:
Was this on purpose? If so, let's just remove it vs. commenting it out and could you explain why you're removing this?

Reply:
Yes, on purpose. Was commented out as constantly produced a PHP error for every single WordPress Administration request (for reference, WordPress adds a .php-error class to the <body> tag in the WordPress Administration when it detects a PHP error).

However, no error is output given the suppression of the function call made, and given that every test checks for the existence of the php-error class, every test would simply fail, providing me with no visibility or coverage as to current Plugin issues.

At the time of writing the tests, I wasn't in a position to clearly determine whether the need to inflate the body response was still needed or could be safely removed. However, this will be removed from 1.9.6 as subsequently I am able to determine the code is not needed.